cterasdk/edge/ftp.py

```diff
@@ -1,20 +1,70 @@
 import logging
 
 from .enum import Mode
+from ..exception import CTERAException
 from .base_command import BaseCommand
 
 
 class FTP(BaseCommand):
     """ Gateway FTP configuration APIs """
 
+    def get_configuration(self):
```

Comment:
Just as an FYI. We should check with Yaniv but I believe FTP was deprecated in Genesis.

Reply:
Since it's already implemented, no harm is keeping it I guess